spec/system/post_revision_spec.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+describe "Post revisions" do
```

Comment:
I think a system spec here is unnecessary given versions are already tested in revisor spec. Also note a single system spec (file) may add a couple of seconds due to startup time, as opposed to a unit test. These things add up fast.

Reply:
Fair, will remove it and replace it with "better coverage" in the unit test